docs/oci-storage.md

```diff
@@ -1,40 +1,152 @@
 # OCI Storage
 
-OCI storage is an experimental feature to store a bundle's resources in an OCI registry, instead of k8s resources.
-Fleet won't be limited by [`etcd` size limitations](https://etcd.io/docs/v3.4/dev-guide/limit/). However,
-Fleet needs write access to a registry and that registry needs to be accessible by the agents in downstream clusters.
+Fleet stores Kubernetes bundle resources in etcd twice by default. This is done using the K8s API. However, etcd has storage limits and is not optimized for large workloads. As your deployments scale, it is more efficient to store these resources externally in an OCI (Open Container Initiative) registry.
```

Comment:
I think mentioning that Fleet stores resources twice is an implementation detail knowledge not strictly needed by the final user.